ignite/cmd/node_query_bank_balances.go

```diff
@@ -53,7 +51,7 @@ func nodeQueryBankBalancesHandler(cmd *cobra.Command, args []string) error {
 
 	var rows [][]string
 	for _, b := range balances {
-		rows = append(rows, []string{fmt.Sprintf("%s", b.Amount), b.Denom})
+		rows = append(rows, []string{b.String()})
```

Comment:
@aljo242 isn't the balance denom still required for the `session.PrintTable()` below to work?

Reply:
You're totally right - thanks !